src/model.ts

```diff
@@ -188,7 +188,8 @@ class ModelClass implements Model, ChangeItem {
     private args: ModelArgs
   ) {}
 
-  addSimpleField(fieldArgs: any): Model {
+  addSimpleField(passedFieldArgs: any): Model {
+    const fieldArgs = { ...passedFieldArgs };
```

Comment:
this makes sure a copy is created that is mutated and the originally passed argument is not mutated